lighthouse-cli/test/fixtures/csp.html

```diff
@@ -0,0 +1,11 @@
+<html>
```

Comment:
license hey oh :)

Reply:
<?php
/**
 * Auto registration for LiteSpeed classes
 *
 * @since      	1.1.0
 * @since  		3.0 Moved into /
 * @package    	LiteSpeed
 * @author     	LiteSpeed Technologies <info@litespeedtech.com>
 */
defined( 'WPINC' ) || exit;

if ( ! function_exists( 'litespeed_autoload' ) ) {
	function litespeed_autoload( $cls )
	{
		if ( strpos( $cls, '.' ) !== false ) {
			return;
		}

		if ( strpos( $cls, 'LiteSpeed' ) !== 0 ) {
			return;
		}

		$file = explode( '\\', $cls );
		array_shift( $file );
		$file = implode( '/', $file );
		$file = str_replace( '_', '-', strtolower( $file ) );

		if ( strpos( $file, 'lib/' ) === 0 || strpos( $file, 'cli/' ) === 0 || strpos( $file, 'thirdparty/' ) === 0 ) {
			$file = LSCWP_DIR . $file . '.cls.php';
		}
		else {
			$file = LSCWP_DIR . 'src/' . $file . '.cls.php';
		}

		if ( file_exists( $file ) ) {
			require_once $file;
		}
	}
}

spl_autoload_register( 'litespeed_autoload' );

Reply:
Registro del sistema
[000:009][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.07.978 a.m.] P[-[GBACrashDetector init] GBACrashDetector.m(123)] 
Last exit was unexpected: 0, with memory warning: 0, with active call: 0
[000:202][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.171 a.m.] P[-[GBAApplicationDelegate application:didFinishLaunchingAfterDevicePromotionsWithOptions:] GBAApplicationDelegate.m(1146)] InDbg-0 RecoveryState-None
[000:258][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.227 a.m.] P[-[GBMAccountManager initWithRoot:fetcherService:] GBMAccountManager.m(170)] MDM integration is enabled.
[000:275][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.244 a.m.] P[-[GBMPhotosStore pruneDatabase] GBMPhotosStore.m(710)] Queried successfully for file store size, 0
[000:285][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.255 a.m.] P[-[GBMRoot application:didFinishLaunchingWithOptions:] GBMRoot.m(1287)] DidFinishLaunching
[000:355][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.324 a.m.] P[-[GBMUserClient initWithRoot:avatarsClient:photosClient:userAccount:fetcherService:] GBMUserClient.m(1142)] GPCPersonCache size:22 executionTime = 0.005 seconds
[000:374][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.344 a.m.] P[-[GBMUserClient initWithRoot:avatarsClient:photosClient:userAccount:fetcherService:] GBMUserClient.m(1142)] GPCPersonCache size:23 executionTime = 0.005 seconds
[000:416][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.385 a.m.] P[-[GBMRoot goOnlineWithUserCompletion:] GBMRoot.m(606)] Bringing root online
[000:416][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.386 a.m.] P[-[GBMRoot goOnlineWithUserCompletion:] GBMRoot.m(609)] 2 clients to bring online
[000:420][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.390 a.m.] P[-[GBMUserNetworkApiClient updateClientIdentifierWithJid:] GBMUserNetworkApiClient.m(142)] Settings jid: martinezramoos@gmail.com/IOS88E0D907CA for account: martinezramoos@gmail.com
[000:422][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.392 a.m.] P[-[GBMUserNetworkApiClient handleAuthenticationResponse:withError:completion:] GBMUserNetworkApiClient.m(96)] Handling auth response with error: NO.
[000:426][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.396 a.m.] P[-[GBMUserNetworkApiClient updateClientIdentifierWithJid:] GBMUserNetworkApiClient.m(142)] Settings jid: 0dimkk0oye488@id.talk.google.com/IOS9C6EB434C7 for account: erickadalberto@icloud.com
[000:427][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.397 a.m.] P[-[GBMUserNetworkApiClient handleAuthenticationResponse:withError:completion:] GBMUserNetworkApiClient.m(96)] Handling auth response with error: NO.
[000:469][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.438 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[000:469][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.439 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[000:470][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.439 a.m.] P[-[GBARootViewControllerBase viewDidLoad] GBARootViewControllerBase.m(243)] Root view did load, presenting main view for user client erickadalberto@icloud.com
[000:470][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.440 a.m.] P[-[GBARootViewControllerBase presentMainViewForUserClient:] GBARootViewControllerBase.m(389)] Presenting main view for user client: erickadalberto@icloud.com
[000:471][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.441 a.m.] P[-[GBARootViewControllerBase presentMainViewForUserClient:additionalLoadingClients:] GBARootViewControllerBase.m(397)] Presenting main view for user client: erickadalberto@icloud.com and 0 additional clients
[000:471][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.441 a.m.] P[-[GBARootViewControllerBase presentMainViewForUserClient:additionalLoadingClients:] GBARootViewControllerBase.m(408)] Presenting loading screen with 1 avatars. 2 clients performing coldstart
[000:473][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.442 a.m.] P[-[GBARootViewControllerBase presentLoadingForUserClient:displayAvatars:actionCompletePromise:] GBARootViewControllerBase.m(342)] Presenting loading screen for user client: erickadalberto@icloud.com with 1 avatars
[000:514][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.483 a.m.] P[-[GBMRoot applicationDidBecomeActive:] GBMRoot.m(1353)] DidBecomeActive
[000:669][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.638 a.m.] P[-[GBARootViewControllerBase root:didAddUserClients:] GBARootViewControllerBase.m(557)] Added 2 user clients
[000:760][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.729 a.m.] P[-[GBARootViewControllerBase presentLoadingForUserClient:displayAvatars:actionCompletePromise:]_block_invoke GBARootViewControllerBase.m(363)] Loading complete, presenting main view for user client: erickadalberto@icloud.com
[000:760][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.730 a.m.] P[-[GBARootViewControllerBase presentMainViewForUserClient:] GBARootViewControllerBase.m(389)] Presenting main view for user client: erickadalberto@icloud.com
[000:761][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.730 a.m.] P[-[GBARootViewControllerBase presentMainViewForUserClient:additionalLoadingClients:] GBARootViewControllerBase.m(397)] Presenting main view for user client: erickadalberto@icloud.com and 0 additional clients
[000:788][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.758 a.m.] P[-[GBARootViewControllerBase presentMainViewForUserClient:additionalLoadingClients:] GBARootViewControllerBase.m(449)] Navigating to conversation list for client erickadalberto@icloud.com
[000:792][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.761 a.m.] P[-[GBAApplicationDelegate onNavigate:] GBAApplicationDelegate.m(1997)] Setting savedUserClient in onNavigate
[000:792][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.762 a.m.] P[-[GBAApplicationDelegate setSavedUserClient:] GBAApplicationDelegate.m(258)] Saving selected user email address : erickadalberto@icloud.com .
[000:793][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.762 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[000:793][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.08.763 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[001:820][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.09.789 a.m.] P[-[GBMUserAuthorizerClient internalFetchSelfInfoForAccount:completion:]_block_invoke GBMUserAuthorizerClient.m(97)] Fetch Info received with error: NO.
[001:827][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.09.796 a.m.] P[-[GBMUserAuthorizerClient internalFetchSelfInfoForAccount:completion:]_block_invoke GBMUserAuthorizerClient.m(97)] Fetch Info received with error: NO.
[002:105][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.074 a.m.] P[-[GBMUserAuthorizerClient internalFetchSelfInfoForAccount:completion:]_block_invoke_2 GBMUserAuthorizerClient.m(124)] Fetch P2P Bits received with error: NO.
[002:198][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.168 a.m.] P[-[GBMUserAuthorizerClient internalFetchSelfInfoForAccount:completion:]_block_invoke_2 GBMUserAuthorizerClient.m(124)] Fetch P2P Bits received with error: NO.
[002:676][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.645 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal] GBMAccountManager.m(592)] Fetching all keychain accounts.
[002:677][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.646 a.m.] P[-[GBAApplicationDelegate onNavigate:] GBAApplicationDelegate.m(1997)] Setting savedUserClient in onNavigate
[002:678][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.647 a.m.] P[-[GBAApplicationDelegate setSavedUserClient:] GBAApplicationDelegate.m(258)] Saving selected user email address : erickadalberto@icloud.com .
[002:679][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.649 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[002:680][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.649 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[002:742][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.711 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[002:742][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.712 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[002:811][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.781 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal]_block_invoke GBMAccountManager.m(597)] Fetched 2 account(s).
[002:812][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.781 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:martinezramoos@gmail.com, gaiaId:107482228871743129773
[002:812][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.782 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[002:813][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.782 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[002:813][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.783 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:erickadalberto@icloud.com, gaiaId:111785307592770443791
[002:814][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.783 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[002:815][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.784 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[002:815][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.785 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:martinezramoos@gmail.com, gaiaID:107482228871743129773 with resultCode 15.
[002:816][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.10.786 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:erickadalberto@icloud.com, gaiaID:111785307592770443791 with resultCode 15.
[004:808][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.12.777 a.m.] P[-[GBAApplicationDelegate onNavigate:] GBAApplicationDelegate.m(1997)] Setting savedUserClient in onNavigate
[004:808][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.12.778 a.m.] P[-[GBAApplicationDelegate setSavedUserClient:] GBAApplicationDelegate.m(258)] Saving selected user email address : erickadalberto@icloud.com .
[004:809][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.12.778 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[004:809][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.12.779 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[007:009][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.14.978 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal] GBMAccountManager.m(592)] Fetching all keychain accounts.
[007:011][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.14.980 a.m.] P[-[GBAApplicationDelegate onNavigate:] GBAApplicationDelegate.m(1997)] Setting savedUserClient in onNavigate
[007:012][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.14.981 a.m.] P[-[GBAApplicationDelegate setSavedUserClient:] GBAApplicationDelegate.m(258)] Saving selected user email address : erickadalberto@icloud.com .
[007:012][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.14.982 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[007:013][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.14.982 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[007:041][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.010 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[007:041][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.010 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[007:085][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.054 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal]_block_invoke GBMAccountManager.m(597)] Fetched 2 account(s).
[007:086][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.055 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:martinezramoos@gmail.com, gaiaId:107482228871743129773
[007:086][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.055 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[007:087][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.056 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[007:087][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.057 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:erickadalberto@icloud.com, gaiaId:111785307592770443791
[007:087][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.057 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[007:088][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.058 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[007:089][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.058 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:martinezramoos@gmail.com, gaiaID:107482228871743129773 with resultCode 15.
[007:089][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.15.059 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:erickadalberto@icloud.com, gaiaID:111785307592770443791 with resultCode 15.
[010:676][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.644 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal] GBMAccountManager.m(592)] Fetching all keychain accounts.
[010:678][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.647 a.m.] P[-[GBAApplicationDelegate onNavigate:] GBAApplicationDelegate.m(1997)] Setting savedUserClient in onNavigate
[010:679][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.648 a.m.] P[-[GBAApplicationDelegate setSavedUserClient:] GBAApplicationDelegate.m(258)] Saving selected user email address : erickadalberto@icloud.com .
[010:682][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.651 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[010:682][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.652 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[010:709][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.678 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[010:709][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.678 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[010:752][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.721 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal]_block_invoke GBMAccountManager.m(597)] Fetched 2 account(s).
[010:752][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.721 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:martinezramoos@gmail.com, gaiaId:107482228871743129773
[010:753][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.722 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[010:753][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.723 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[010:754][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.723 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:erickadalberto@icloud.com, gaiaId:111785307592770443791
[010:754][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.724 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[010:755][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.725 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[010:756][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.725 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:martinezramoos@gmail.com, gaiaID:107482228871743129773 with resultCode 15.
[010:757][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.18.726 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:erickadalberto@icloud.com, gaiaID:111785307592770443791 with resultCode 15.
[014:776][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.745 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal] GBMAccountManager.m(592)] Fetching all keychain accounts.
[014:778][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.747 a.m.] P[-[GBAApplicationDelegate onNavigate:] GBAApplicationDelegate.m(1997)] Setting savedUserClient in onNavigate
[014:779][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.748 a.m.] P[-[GBAApplicationDelegate setSavedUserClient:] GBAApplicationDelegate.m(258)] Saving selected user email address : erickadalberto@icloud.com .
[014:780][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.749 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[014:781][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.750 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[014:807][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.776 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[014:807][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.777 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[014:852][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.820 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal]_block_invoke GBMAccountManager.m(597)] Fetched 2 account(s).
[014:852][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.821 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:martinezramoos@gmail.com, gaiaId:107482228871743129773
[014:852][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.822 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[014:853][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.822 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[014:854][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.823 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:erickadalberto@icloud.com, gaiaId:111785307592770443791
[014:854][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.823 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[014:855][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.824 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[014:856][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.825 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:martinezramoos@gmail.com, gaiaID:107482228871743129773 with resultCode 15.
[014:856][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.22.826 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:erickadalberto@icloud.com, gaiaID:111785307592770443791 with resultCode 15.
[018:805][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.774 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal] GBMAccountManager.m(592)] Fetching all keychain accounts.
[018:806][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.775 a.m.] P[-[GBAApplicationDelegate onNavigate:] GBAApplicationDelegate.m(1997)] Setting savedUserClient in onNavigate
[018:807][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.776 a.m.] P[-[GBAApplicationDelegate setSavedUserClient:] GBAApplicationDelegate.m(258)] Saving selected user email address : erickadalberto@icloud.com .
[018:807][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.776 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[018:807][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.776 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[018:830][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.798 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[018:830][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.799 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[018:872][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.840 a.m.] P[-[GBMAccountManager fetchAndValidateAllOOBClientsInternal]_block_invoke GBMAccountManager.m(597)] Fetched 2 account(s).
[018:874][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.842 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:martinezramoos@gmail.com, gaiaId:107482228871743129773
[018:874][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.843 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[018:875][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.844 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[018:875][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.844 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:] GBMAccountManager.m(638)] Validating account email:erickadalberto@icloud.com, gaiaId:111785307592770443791
[018:875][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.844 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(764)] Validating authentication and selfinfo for account.
[018:876][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.845 a.m.] P[-[GBMRoot validateOAuthAndSelfInfoForEmailAddress:gaiaId:] GBMRoot.m(768)] User account already logged in.
[018:877][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.846 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:martinezramoos@gmail.com, gaiaID:107482228871743129773 with resultCode 15.
[018:877][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.26.846 a.m.] P[-[GBMAccountManager validateAccountForEmailAddress:displayName:gaiaId:]_block_invoke GBMAccountManager.m(663)] Validation complete for account email:erickadalberto@icloud.com, gaiaID:111785307592770443791 with resultCode 15.
[020:309][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.28.278 a.m.] P[-[GBAApplicationDelegate onNavigate:] GBAApplicationDelegate.m(1997)] Setting savedUserClient in onNavigate
[020:310][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.28.279 a.m.] P[-[GBAApplicationDelegate setSavedUserClient:] GBAApplicationDelegate.m(258)] Saving selected user email address : erickadalberto@icloud.com .
[020:311][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.28.279 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(242)] Selected user email address : erickadalberto@icloud.com .
[020:311][1031] (GBCDiagnosticsPrivate.mm:33): [2021-04-22 12.22.28.280 a.m.] P[-[GBAApplicationDelegate savedUserClient] GBAApplicationDelegate.m(252)] Using user client for account Erick Martinez <erickadalberto@icloud.com>.
[022:920][1031] (Hangouts:GHKLoggingManagerHelper.mm:137): Error enumerating logs directory [/var/mobile/Containers/Data/Application/DF5E7974-F0CC-46C1-A3AC-94D4F52B8EFA/Library/Caches/hangoutskit.diagnostic.logs/111785307592770443791]: Error Domain=NSCocoaErrorDomain Code=260 "La carpeta “111785307592770443791” no existe." UserInfo={NSUserStringVariant=(
    Folder
), NSFilePath=/var/mobile/Containers/Data/Application/DF5E7974-F0CC-46C1-A3AC-94D4F52B8EFA/Library/Caches/hangoutskit.diagnostic.logs/111785307592770443791, NSUnderlyingError=0x282535710 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
[024:525][1031] (Hangouts:GHKLoggingManagerHelper.mm:137): Error enumerating logs directory [/var/mobile/Containers/Data/Application/DF5E7974-F0CC-46C1-A3AC-94D4F52B8EFA/Library/Caches/hangoutskit.diagnostic.logs/111785307592770443791]: Error Domain=NSCocoaErrorDomain Code=260 "La carpeta “111785307592770443791” no existe." UserInfo={NSUserStringVariant=(
    Folder
), NSFilePath=/var/mobile/Containers/Data/Application/DF5E7974-F0CC-46C1-A3AC-94D4F52B8EFA/Library/Caches/hangoutskit.diagnostic.logs/111785307592770443791, NSUnderlyingError=0x282553c00 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
[030:175][1031] (Hangouts:GHKLoggingManagerHelper.mm:137): Error enumerating logs directory [/var/mobile/Containers/Data/Application/DF5E7974-F0CC-46C1-A3AC-94D4F52B8EFA/Library/Caches/hangoutskit.diagnostic.logs/111785307592770443791]: Error Domain=NSCocoaErrorDomain Code=260 "La carpeta “111785307592770443791” no existe." UserInfo={NSUserStringVariant=(
    Folder
), NSFilePath=/var/mobile/Containers/Data/Application/DF5E7974-F0CC-46C1-A3AC-94D4F52B8EFA/Library/Caches/hangoutskit.diagnostic.logs/111785307592770443791, NSUnderlyingError=0x282512670 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}